warehouse/manage/views/organizations.py

```diff
@@ -246,6 +247,86 @@ def create_organization_application(self):
         return HTTPSeeOther(self.request.path)
 
 
+@view_defaults(
+    route_name="manage.organizations.application",
+    context=OrganizationApplication,
+    renderer="manage/organization/application.html",
+    uses_session=True,
+    require_csrf=True,
+    require_methods=False,
+    permission=Permissions.OrganizationApplicationsManage,
+    has_translations=True,
+)
+class ManageOrganizationApplicationViews:
+    def __init__(self, organization_application, request):
+        self.organization_application = organization_application
+        self.request = request
+
+    @view_config(
+        request_method="GET", permission=Permissions.OrganizationApplicationsManage
```

Comment:
Is setting the same permission on the view_config as the view_defaults required? Both views use the same permission right now. Removing the duplication has the `@view_config` fit on a single line and looks nicer to me.

Reply:
96aacc53b52d16151c2c9564714ef3350904b292